From 00fd73cceea6d53e511044eae413e614de011912 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:22:55 -0400
Subject: [cpufreq] governor: Remove fossil comment in the cpufreq_governor_dbs()

Message-id: <1409235879-1283-82-git-send-email-prarit@redhat.com>
Patchwork-id: 89400
O-Subject: [RHEL7.1 PATCH BZ 1134639 081/185] cpufreq: governor: Remove fossil comment in the cpufreq_governor_dbs()
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit aae467c79b14db0d286764ed9ddbaefe3715ebd2
Author: lan,Tianyu <tianyu.lan@intel.com>
Date:   Fri Nov 15 13:54:01 2013 +0800

    cpufreq: governor: Remove fossil comment in the cpufreq_governor_dbs()

    The related code has been changed and the comment is out of date.
    So remove it.

    Signed-off-by: Lan Tianyu <tianyu.lan@intel.com>
    Acked-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq_governor.c b/drivers/cpufreq/cpufreq_governor.c
index 0806c31..e6be635 100644
--- a/drivers/cpufreq/cpufreq_governor.c
+++ b/drivers/cpufreq/cpufreq_governor.c
@@ -328,10 +328,6 @@ int cpufreq_governor_dbs(struct cpufreq_policy *policy,
           dbs_data->cdata->gov_dbs_timer);
   }
 
-  /*
-   * conservative does not implement micro like ondemand
-   * governor, thus we are bound to jiffes/HZ
-   */
   if (dbs_data->cdata->governor == GOV_CONSERVATIVE) {
    cs_dbs_info->down_skip = 0;
    cs_dbs_info->enable = 1;
-- 
1.7.1